examples/datasource-logs/src/components/QueryEditor.tsx

```diff
@@ -18,10 +18,16 @@ export function QueryEditor({ query, onChange }: Props) {
   return (
     <Stack gap={0}>
       <InlineField label="Query" labelWidth={20} tooltip="Query text">
-        <Input onChange={onQueryTextChange} value={query.queryText || ''} width={70} />
+        <Input id="query-editor-query-text" onChange={onQueryTextChange} value={query.queryText || ''} width={70} />
```

Comment:
I see you are using `test-id` in the other query editor tests but `id` here. Can we keep it consistent? I think the `test-id` is the best way to go